keyboards/handwired/p60/readme.md

```diff
@@ -0,0 +1,20 @@
+# P60
+
+[![](https://user-images.githubusercontent.com/26496/108596854-ee41c900-7387-11eb-8225-e4faf72aa793.png)](https://user-images.githubusercontent.com/26496/108102379-c3562d00-7088-11eb-9083-9921497429ca.JPG)
```

Comment:
```suggestion
![](https://i.imgur.com/7AfyZkkl.jpg)
```

Reply:
Doesn't matter which hosting service I use, GitHub is fine.

Reply:
The image is way too big, though.

Reply:
The embedded image is the one you resized, it's linking to the full-size one. Look at the rich diff.

Reply:
It's *linking* to the thumbnail-sized one, but the image is still the full size.